archive-cleaner

```diff
@@ -68,32 +67,48 @@ def get_packages_from_buildinfo(archive, filename, archive_entry, arch):
     packages = []
     if buildinfo_format > 0:
         for package in installed:
-            pkgfilename = f'{package}.pkg.tar.xz'
+            pkgfilenames = [f'{package}.pkg{ext}' for ext in PKG_EXTENSIONS]
             pkgname = '-'.join(package.split('-')[:-3])
             if not pkgname:
                 print(f'ERROR: failed to parse {package} with format {buildinfo_format} needed for {filename}', file=stderr)
                 continue
-            directory = join(archive, 'packages', pkgname[0], pkgname)
-            path = join(directory, pkgfilename)
-            if not isfile(path):
-                print(f'ERROR: {package} needed for {filename} not in archive {path}', file=stderr)
+            directory = join(archive, 'packages'. pkgname[0], pkgname)
```

Comment:
this line shouldn't change, whats that dot doing there?

Reply:
Yeah that's a typo